### DISCUSSION TITLE
Welcome to catanatron Discussions!

### DISCUSSION THREAD

**Discussion by bcollazo** [bcollazo/catanatron#55] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by braydenrudisill** [comment]
Hello! Sorry unsure how actively this is still being maintained, but it looked really fascinating to me and I was looking to at least run the Open-AI Gym Environment. Unfortunately when I run `gym.make("catanatron_gym:catanatron-v0")` it hits some KeyError saying it can't find the key `'render_modes'`. Any help would be appreciated if you can reproduce, mine should be a fairly clean install. Also if there is a discord for this, we could chat more, cheers!

Here is the code causing the issue and my pip installs below
```
import random
import gym
env = gym.make("catanatron_gym:catanatron-v0")
```

<img width="147" alt="image" src="https://user-images.githubusercontent.com/55212800/179379223-c4b0b8ec-019c-42e9-958f-db158b2ea9be.png">

**Reply by bcollazo** [reply]
Hey! Consider it actively maintained; happy to help you use it.

You are right; the PyPi versions that were published (3.1.2 and 3.0.1) were slightly outdated and were actually broken with the latest gym (0.25.0). I just merged a fix (see https://github.com/bcollazo/catanatron/pull/192) and published new versions to PyPi. If you try again a fresh install, you should get `catanatron==3.2.1` and `catanatron-gym==3.2.1` which should work. Can you give it a try?

Also, for even more functionality (CLI and visualizing games) you may want to clone the entire repository and use from source. [These instructions](https://github.com/bcollazo/catanatron#installation) should do the trick.

**Comment by paulxiep** [comment]
Hello, there's some cool engine you've built here.

I've been writing a Catan game in Python myself since 15 days ago.

I have a basic rule-based AI that works on mine that I'd like to port to your engine, but I'm finding it overwhelming to figure out how to get info from all your game states, game board and possible actions. I'm hoping you can provide a more detailed guidelines. It's not easy tracing all your Game, Board, Map, and Action files myself. I mean, I'm running the decide method that's printing various things out right about now, but that'd surely take a long while before I figure out everything, not sure if I'll be able to stick to it.
